sample_files/.travis.yml

```diff
@@ -11,9 +11,21 @@ addons:
 
 env:
   global:
-  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
```

Comment:
Agree with @yvaucher. If I take as an example the connector, we would have
```
  matrix:
  - LINT_CHECK="1"
  - TESTS="1" ODOO_REPO="odoo/odoo"
  - TESTS="1" ODOO_REPO="OCA/OCB"
  - DOCS="1"
  - TRANSIFEX="1"
  global:
  - secure: ...
  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0" DOCS="0"
```
which seems way cleaner than repeating `TEST="0"` on every lines